README.md

```diff
@@ -18,20 +18,14 @@ gem 'aws-sdk-rails', '~> 3'
 
 This gem also brings in the following AWS gems:
 
+* `aws-sdk-s3`
 * `aws-sdk-ses`
 * `aws-sdk-sesv2`
 * `aws-sdk-sqs`
+* `aws-sdk-sns`
 * `aws-record`
 * `aws-sessionstore-dynamodb`
 
-If you want to use other services (such as S3), you will still need to add them to your
```

Comment:
this wording probably needs to be retained, but another ex is needed. I'm not sure if we actually need it tho or if it's self explanatory

Reply:
This comment is a general AWS SDK explanation and how aws-sdk-rails does not bring in all of the SDK. V2 SDK was "monolithic" such that gem aws-sdk would install everything. It's probably fine to remove at this point in v3's lifecycle.